### PR TITLE
Issue 37991: Allow metadata override for URLs of custom query  

### PR DESCRIPTION
**Pull request by ilikepies101**
Custom queries allow for insert, edit, and delete. This change is limited only to custom queries.

### CODE REVIEW

File: api/src/org/labkey/api/data/AbstractTableInfo.java [hunk 1]
```diff
@@ -1778,19 +1780,19 @@ public String toMemTrackerString()
     @Override
     public boolean hasXMLInsertOverride()
```
Comment (labkey-kevink) [inline api/src/org/labkey/api/data/AbstractTableInfo.java]: Please rename the methods as well to match the field names.

Comment (ilikepies101) [reply api/src/org/labkey/api/data/AbstractTableInfo.java]: Oh shoot I thought I did a replace all. Good find.

Comment (ilikepies101) [reply api/src/org/labkey/api/data/AbstractTableInfo.java]: Sounds good

Comment (ilikepies101) [reply api/src/org/labkey/api/data/AbstractTableInfo.java]: Should be done now.

**Pull request merged**